src/app/features/genes/components/gene-resources/gene-resources.component.ts

```diff
@@ -1,38 +1,23 @@
-import { Component, Input } from '@angular/core';
+import { Component, Input, OnInit } from '@angular/core';
 
-import { Gene } from '../../../../models';
-
-interface AdditionalResource {
-  title: string;
-  description: string;
-  linkText: string;
-  link: string;
-}
+import { Gene, AdditionalResource } from '../../../../models';
 
 @Component({
   selector: 'gene-resources',
   templateUrl: './gene-resources.component.html',
   styleUrls: ['./gene-resources.component.scss'],
 })
-export class GeneResourcesComponent {
-  _gene: Gene | undefined;
-  get gene(): Gene | undefined {
-    return this._gene;
-  }
-  @Input() set gene(gene: Gene | undefined) {
-    this._gene = gene;
-    this.init();
-  }
-  additionalResources: AdditionalResource[] = [] as AdditionalResource[];
+export class GeneResourcesComponent implements OnInit {
+  @Input() gene: Gene | undefined;
+  
+  additionalResources: AdditionalResource[] = [];
 
-  reset() {
-    this.additionalResources = [];
+  ngOnInit(): void {
+    this.init();
```

Comment:
moved code to ngOnInit() as using the setter to update the static content on changes didn't seem to be needed.